src/jquery-sticky-header-footer.js

```diff
@@ -48,7 +48,8 @@
                 @method tearDown
              */
             tearDown: function() {
-                var element = this.first();
```

Comment:
Yes, this is meant to be a jquery object. `this` was meant to be the plugin itself.